### PR TITLE
Recreated faculty fields branch

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds the ACF Fields for People and additional fields for when the `template-faculty.php` template is used. In addition, the people custom post type's slug is updated to point to `/faculty/`.

**Motivation and Context**
We want to be able to reuse the people custom post type to serve up various types of faculty profiles.

**How Has This Been Tested?**
Changes are available for review in DEV.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [x] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by rjucf — APPROVED** [review]

File: dev/acf-export.json [hunk 1]
```diff
@@ -2299,6 +2299,355 @@
...
+                },
+                "placement": "top",
+                "endpoint": 0
+            },
+            {
+                "key": "field_60afe850124b6",
+                "label": "College",
+                "name": "person_college",
+                "type": "taxonomy",
+                "instructions": "Choose the college the faculty member is associate with.",
```
Comment (cjg89) [inline dev/acf-export.json:2327]: Typo: "associate" should be "associated"

File: dev/acf-export.json [hunk 2]
```diff
@@ -2699,6 +3048,104 @@
...
+    {
+        "key": "group_60afe4968b8d2",
+        "title": "Person Fields",
+        "fields": [
+            {
+                "key": "field_60afe4f815aac",
+                "label": "Title",
+                "name": "person_title",
+                "type": "text",
+                "instructions": "The title of the person",
```
Comment (cjg89) [inline dev/acf-export.json:3060]: Maybe use "The person's job title" here instead?

File: dev/acf-export.json [hunk 3]
```diff
@@ -2699,6 +3048,104 @@
...
+                "prepend": "",
+                "append": "",
+                "maxlength": ""
+            },
+            {
+                "key": "field_60afe51815aad",
+                "label": "Email",
+                "name": "person_email",
+                "type": "email",
+                "instructions": "The email of the person.",
```
Comment (cjg89) [inline dev/acf-export.json:3079]: Maybe use "The person's email address" here instead?

File: includes/people-functions.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Responsible for People custom post type functionality
+ */
+
+/**
+ * Modifies the rewrite rules for the People
+ * custom post type
+ * @author Jim Barnes
+ * @since 3.9.3
```
Comment (cjg89) [inline includes/people-functions.php:10]: `since` will need to be bumped to 3.10.0

**Review by cjg89 — APPROVED** [review]

**Pull request merged**